### ISSUE TITLE
大佬，您也是先写在word里，然后转latex吗

### ISSUE THREAD

**Issue by AlexShyXie** [yujmo/Mycitavi#1]
和别人合作需要word，但是论文投稿加主文档加其他其他word，会有七八个文档，我想转成latex进行审查和投稿，不知道方不方便

**Comment by yujmo** [comment]
是的。我也是先写在word里面，然后用pan2doc这样的工具，将word转成latex。  我自定义的样式，就是可以直接在word中插入latex cite论文时候的key。

多人合作的话，用word是非常好用的，word中有一个叫做合并的功能，类似于git merge，可以把其他人的文档直接合并过来。

**Comment by AlexShyXie** [comment]
> 是的。我也是先写在word里面，然后用pan2doc这样的工具，将word转成latex。  我自定义的样式，就是可以直接在word中插入latex cite论文时候的key。
> 
> 多人合作的话，用word是非常好用的，word中有一个叫做合并的功能，类似于git merge，可以把其他人的文档直接合并过来。

那，对于title page，还有正文中的图片，表格，我们都是单独存放成word和tif，word里没有做交叉引用，你们是不是也是手动合并到latex里呢？

**Comment by yujmo** [comment]
正文中的图片、表格，我们是手动合并到latex的。
因为图、表这些数量不会太多。可能就二十多个，手动合并问题不大。
大头还是参考文献。


导入citavi文献，和在正文中引用图、表，他们在word中的域代码不一致，因为他们属于两个不同的东西。 我还没研究过呢。

**Comment by AlexShyXie** [comment]
> 正文中的图片、表格，我们是手动合并到latex的。
> 因为图、表这些数量不会太多。可能就二十多个，手动合并问题不大。
> 大头还是参考文献。
> 
> 
> 导入citavi文献，和在正文中引用图、表，他们在word中的域代码不一致，因为他们属于两个不同的东西。 我还没研究过呢。
> 

了解了，谢谢
